adoc/SA_vars.adoc

```diff
@@ -3,9 +3,10 @@
 :title: SUSE SAP automation solution
 
 
-:author: Peter Schinagl 
-:authorEmail: peters@suse.com
-
+//:author: Peter Schinagl 
+//:authorEmail: peters@suse.com
+:author: Abdelrahman Mohamed 
+:authorEmail: amohamed@suse.com
```

Comment:
Please dont change the SA_vars.adoc file too much as it would create a endless loop of changes due to switching clouds.
Btw. we are ALL mentioned as authors - no need to change it

Reply:
Sorry, I was just trying to discover how to produce PDF version. Where is the part that contains our names?